bcsaba/CurrencyExchange
# Added current exchange rate display to UI. Added UI part of saving selected rate.

### Purpose

This PR is about adding the UI component to display the current exchange rate queried from the MNB site and make it possible for the users to initiate saving a selected exchange rate into the local database. (Back-end support needs to be added.)

### What is changed

A new item was added to the navigation menu to get to the current exchange rates display.
The exchange rate listing page shows the list of currencies with their actual value. It also provides a Save button for each item.
Pressing the button the user can specify a comment for saving the selected exchange rate into the database.
Also some simple toast notifications were added for better UX.

### How to test

1. Use the new navigation item to get to the listing page. The list should be displayed in a grid format with buttons to make the save of the items possible.
2. Pressing the save button a modal dialog should be displayed
3. Cancelling the modal does nothing.
4. Pressing the Save button in the model, should initiate the save for the back-end. It will fail currently because the lack of back-end functionality. The failure should be indicated with a Toast message.

<details>
<summary>Screenshots</summary>

![image](https://github.com/bcsaba/CurrencyExchange/assets/3913211/b7dc7905-140d-4696-8a10-2f198a6861f1)

![image](https://github.com/bcsaba/CurrencyExchange/assets/3913211/d98c83c4-29db-4ffa-ab56-ae6496800f70)

![image](https://github.com/bcsaba/CurrencyExchange/assets/3913211/04f718fe-644a-4666-8923-6d89a38bebc1)


</details>
